ClosedXML/Excel/CalcEngine/Functions/Text.cs

```diff
@@ -231,7 +231,7 @@ private static object Search(List<Expression> p)
             var search = WildcardToRegex(p[0]);
             var text = (string)p[1];
 
-            if ("" == text) throw new ArgumentException("Invalid input string.");
+            if (string.IsNullOrEmpty(text)) throw new ArgumentException("Invalid input string.");
```

Comment:
is it safe to handle null with an exception too or was that an oversight?

Reply:
the lchanges below in this file are similiar

Reply:
Null would cause a exception in line 243. It is safe.